packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/hooks/useReviewWorkflows.js

```diff
@@ -30,25 +29,10 @@ export function useReviewWorkflows(workflowId) {
     return data;
   }
 
-  async function updateEntityStage({ entityId, stageId }) {
-    const {
-      data: { data },
-      // TODO: endpoint differs for collection-types and single-types
-    } = await put(`${API_CM_BASE_URL}/collection-types/:model_uid/${entityId}/stage`, {
-      data: { id: stageId },
-    });
-
-    return data;
-  }
-
   function updateWorkflowStages(workflowId, stages) {
```

Comment:
I am convinced now this mutation should also be moved out of the hook and be inlined into the settings page. Will do that in a follow-up PR.

Reply:
Wouldn't that leave this hook almost pointless? 🤔  what's the thinking behind all the moving of the hooks?

Reply:
I think the hook could remain, but it will only be responsible to fetch workflows, which we need in two places (settings, edit-view). That makes it very easy to read and test.

Reply:
How come you want to omit the mutations though? Anyone wanting to use this hook in a different part of the app would have to write their own mutations? 🤔

Reply:
As soon as one of the mutations is needed somewhere else, I'm happy to move them back in, but right now they are only needed in one place each. For the moment that makes everything much easier to read and follow ...